core/pubnub_advanced_history.c

```diff
@@ -82,8 +84,8 @@ enum pubnub_res pubnub_message_counts(pubnub_t*   pb,
 }
 
 
-int pubnub_get_chan_msg_counts(pubnub_t* pb, 
-                               size_t* io_count, 
+int pubnub_get_chan_msg_counts(pubnub_t*                     pb,
+                               size_t*                       io_count,
```

Comment:
strange formatter ;o

Reply:
@Xavrax It aligns argument names, and the next line has a pretty long type, so it added that much to the other lines.
This is the default CLion formatter behavior.